Tasks/PublishPipelineArtifactV1/task.json

```diff
@@ -0,0 +1,44 @@
+{
+    "id": "ECDC45F6-832D-4AD9-B52B-EE49E94659BE",
+    "name": "PublishPipelineArtifact",
+    "friendlyName": "Publish pipeline artifact",
+    "description": "Publish a local directory or file as a named artifact for the current pipeline",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/utility/publish-pipeline-artifact",
+    "helpMarkDown": "Publish a local directory or file as a named artifact for the current pipeline.",
+    "category": "Utility",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 1,
+        "Minor": 0,
+        "Patch": 0
+    },
+    "groups": [],
+    "demands": [],
+    "preview": true,
+    "minimumAgentVersion": "2.140.1",
+    "inputs": [
+        {
+            "name": "artifactName",
+            "type": "string",
+            "label": "The name of this artifact",
+            "defaultValue": "drop",
+            "required": true,
+            "helpMarkDown": "The name of this artifact."
+        },
+        {
+            "name": "targetPath",
+            "type": "filePath",
+            "label": "Path to publish",
+            "defaultValue": "",
+            "required": true,
+            "helpMarkDown": "The folder or file path to publish. This can be a fully-qualified path or a path relative to the root of the repository. Wildcards are not supported. [Variables](https://go.microsoft.com/fwlink/?LinkID=550988) are supported."
```

Comment:
Reword: `The path of the file or directory to publish. Can be absolute or relative to the default working directory. Can include [variables](https://go.microsoft.com/fwlink/?LinkID=550988), but wildcards are not supported.`

@jahquin-msft -- relative paths are resolved against `$(System.DefaultWorkingDirectory)`, correct? If not, what does the task currently resolve against?

Reply:
See https://github.com/microsoft/azure-pipelines-agent/pull/2279/files#r289164137

I'm fairly sure that `Path.GetFullPath(targetPath)` (in the agent plugin) get resolved against the current working directory for the agent (e.g. c:/agent/_work) since this logic executes in the agent process (unlike other tasks).

Reply:
I added some examples showing what I would expect different `path` values to resolve as. https://github.com/microsoft/azure-pipelines-agent/pull/2279#discussion_r289368171